bundles/WebToPrintBundle/translations/admin.en.yaml

```
@@ -54,5 +54,13 @@ web2print_wkhtmltopdf_settings: Wkhtmltopdf Settings
 web2print_wkhtmltopdf_binary: wkhtmltopdf Binary
 web2print_wkhtmltopdf_options: Options
 web2print_wkhtmltopdf_options_txt: One per line with '--' and whitespace between key and value (e.g. --key value)
+web2print_gotenberg_settings: Gotenberg Settings
+web2print_gotenberg_documentation: Option Documentation
+web2print_gotenberg_documentation_additions: Warning
+web2print_gotenberg_documentation_additions_text: >-
+    Since Gotenberg is Docker-powered, to make it work across with Pimcore, it would be required to set
+    base url in the print templates and the value should be the same as the web server service<br>
+    eg. <b>&lt;base href=&quot;http://nginx:80/&quot; target=&quot;_blank&quot;&gt;</b>
```

```suggestion
    Since Gotenberg is Docker-powered service, to make it work with Pimcore within the Docker Container, it may be required to set
    base url in the print templates and the value should be the same as the web server service<br>
    eg. <b>&lt;base href=&quot;http://nginx:80/&quot; &gt;</b>
```
Is this clearer? not sure if i am using the docker terminology right
removed target `_blank` as default